src/ngMock/angular-mocks.js

```diff
@@ -1341,13 +1341,13 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
       // service the ngMockE2E version can pass through to the real backend
       originalHttpBackend = $delegate.$$originalHttpBackend || $delegate;
 
-  function createResponse(status, data, headers, statusText) {
+  function createResponse(status, data, headers, statusText, xhrStatus) {
     if (angular.isFunction(status)) return status;
 
     return function() {
       return angular.isNumber(status)
-          ? [status, data, headers, statusText]
-          : [200, status, data, headers];
+          ? [status, data, headers, statusText, xhrStatus || (status === 200 ? 'success' : 'error')]
```

Comment:
@gkalpak It feels abit odd to be calculating xhrStatus (the above example isn't what it should be yet). I went down the path of an hardcoded 'success' value because of the hardcoded `200` on line 1350. Hardcoding 'success' made me wonder if we should be calculating xhrStatus based on status (if xhrStatus was not provided). It brings quiet alot of complexity. 

Maybe it's a good idea to not set any xhrStatus based on the status, apart from line 1350, and change this line (1349) to `[status, data, headers, statusText, xhrStatus]` but this doesn't sound very consistent ...